### PR TITLE
Create USBYara.yaml

### PR DESCRIPTION
**Pull request by mgreen27**
Cool example of monitoring artifact leveraging enrichment via VQL.
image

### CODE REVIEW

File: content/exchange/artifacts/USBYara.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                    $s9 = "Xayemarlwin-pc" wide nocase 
+            
+               condition:
+                    int16(0) == 0x004c and any of them
+            }
+            
+sources:
+  - query: |
+      SELECT * FROM foreach(
+            row={ SELECT DriveName,TimeCreated FROM Artifact.Windows.Monitor.USBPlugIn() },
```
Comment (scudette) [inline content/exchange/artifacts/USBYara.yaml]: So we do have problems importing other exchange artifacts because the import adds a prefix.

Comment (mgreen27) [reply content/exchange/artifacts/USBYara.yaml]: Thats a good point. I have been importing using "Windows." as a prefix and didnt even think about that. Do you think we should add "Exchange." here or add the USBMonitor to the main repo?

Comment (scudette) [reply content/exchange/artifacts/USBYara.yaml]: I think it is better to add Exchange here - but we do need to think about it - the import artifact allows us to change the prefix so maybe we also need to update internal references. 

In any case in the short term adding an exchange prefix here will make it all work out of the box :-)

Comment (mgreen27) [reply content/exchange/artifacts/USBYara.yaml]: just updated :)

**Review by scudette — APPROVED** [review]

**Pull request merged**